tests/cdi/cdi_suite_test.go

```diff
@@ -0,0 +1,36 @@
+package cdi_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	ktests "kubevirt.io/kubevirt/tests"
+	_ "kubevirt.io/kubevirt/tests"
+)
+
+func TestCDI(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "CDI Suite")
+}
+
+var _ = BeforeSuite(func() {
+	out, err := ktests.RunOcCommand("get", "projects")
+	Expect(err).ToNot(HaveOccurred())
+	Expect(out).Should(ContainSubstring("golden-images"))
+
+	_, err = ktests.RunOcCommand("project", "golden-images")
+	Expect(err).ToNot(HaveOccurred())
+})
+
+var _ = AfterSuite(func() {
+	_, err := ktests.RunOcCommand("delete", "pvc", pvcName)
+	Expect(err).ToNot(HaveOccurred())
```

Comment:
If an error occurs here you will not clean up the rest of the stuff.  Would it be better to create your own project to hold the pvc pod and vmi and then for cleanup you can just delete the project.

Reply:
Agree with Adam ^^.  Kubernetes e2e test suites run in their own namespace which is random-ized.